realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -1697,6 +1700,70 @@ public static boolean compactRealm(RealmConfiguration configuration) {
         return BaseRealm.compactRealm(configuration);
     }
 
+    /**
+     * Cancel a named subscription that was created by calling
+     * {@link RealmQuery#findAllAsync(String)}. If after this, some objects are no longer part of
+     * any active  subscription they will be removed from the device (but not deleted on the server).
```

Comment:
they will be removed **locally** from the device?

Reply:
_stick to delete or remove in the same sentence_

but not **removed** on the server

Reply:
I was actually going for a distinction between deleting (which is permanent) vs. remove (which is not), but maybe that is more confusing than helpful. I'll reword